src/main/java/com/example/weatherwaveapi/model/response/weatherapi/weather/Weather.java

```diff
@@ -0,0 +1,9 @@
+package com.example.weatherwaveapi.model.response.weatherapi.weather;
+
+public record Weather(
+        Integer id,
+        String main,
```

Comment:
What "main" field used for? I suggest to rename it into something more meaningful